sci-misc/boinc/boinc-7.6.25.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Am Dienstag, 15. März 2016, 06:11:57 CET schrieb David Seifert:
> > +EAPI=5
> 
> We prefer to only have `EAPI=6` for new code.

Unfortunately wxwidgets.eclass dies then, so I have to stay with EAPI=5 for now until the eclass is upgraded.

 * ERROR: sci-misc/boinc-7.6.31::gentoo failed (depend phase):
 *   EAPI=6 is not supported
 * 
 * Call stack:
 *             ebuild.sh, line 624:  Called source '/home/sed/pryde/PrydeWorX/gentoo_myfork/sci-misc/boinc/boinc-7.6.31.ebuild'
 *   boinc-7.6.31.ebuild, line   7:  Called inherit 'eutils' 'flag-o-matic' 'systemd' 'user' 'versionator' 'wxwidgets'
 *             ebuild.sh, line 320:  Called __qa_source '/home/sed/pryde/PrydeWorX/gentoo_myfork/eclass/wxwidgets.eclass'
 *             ebuild.sh, line 120:  Called source '/home/sed/pryde/PrydeWorX/gentoo_myfork/eclass/wxwidgets.eclass'
 *      wxwidgets.eclass, line  68:  Called die
 * The specific snippet of code:
 *              die "EAPI=${EAPI:-0} is not supported"